bitcoinspv/types/supported_backends.go

```diff
@@ -0,0 +1,27 @@
+package types
+
+type (
+	SupportedBtcNetwork string
+	SupportedBtcBackend string
+)
+
+const (
+	BtcMainnet SupportedBtcNetwork = "mainnet"
```

Comment:
Lets rename it to just `Mainnet` etc. and put it inside a package `btc` so then we call from different places `types.btc.Mainnet`.